src/WarcraftLegacies.Source/Quests/Legion/QuestLegionCaptureSunwell.cs

```diff
@@ -22,7 +22,7 @@ public sealed class QuestLegionCaptureSunwell : QuestData
       "The Sunwell has been captured by the Scourge. It now writhes with necromantic energy.";
```

Comment:
Not sure when it was changed but the entire flavour of this quest is nonsense. Dreadlords aren't Necromancers so what does writhing with necromantic energy have to do with them getting Astral Walk?

Reply:
I didn't make the flavor. But I can try to rewrite it so it makes sense

Reply:
I wanted to play on the idea that the sunwell is kinda Ike the well of eternity. So it gives demon a passageway type stuff. Energy of the sun can be used to travel between worlds

Reply:
Now that you point it out, I think I wrote it... Ok, let's mark this as resolved and I'll fix it in another PR: https://github.com/AzerothWarsLR/WarcraftLegacies/issues/2471